cns/middlewares/k8sSwiftV2.go

```diff
@@ -194,19 +212,27 @@ func (k *K8sSWIFTv2Middleware) getIPConfig(ctx context.Context, podInfo cns.PodI
 			default:
 				nicType = cns.DelegatedVMNIC
 			}
-
-			podIPInfos[i] = cns.PodIpInfo{
-				PodIPConfig: cns.IPSubnet{
-					IPAddress:    ip,
-					PrefixLength: uint8(prefixSize),
-				},
-				MacAddress:        interfaceInfo.MacAddress,
-				NICType:           nicType,
-				SkipDefaultRoutes: false,
-				HostPrimaryIPInfo: cns.HostIPInfo{
-					Gateway: interfaceInfo.GatewayIP,
-				},
+			if nicType != cns.BackendNIC {
```

Comment:
We have added a new filed, not sure why it's done for the same value. Let me use the new one. I think you meant we should use cns.NodeNetworkInterfaceBackendNIC right? The other will still work since it's basically string comparison, but i will change it to avoid confusions.